packages/docusaurus-plugin-content-docs/src/slug.ts

```diff
@@ -21,14 +21,14 @@ import type {NumberPrefixParser} from '@docusaurus/plugin-content-docs';
 
 export default function getSlug({
   baseID,
-  frontmatterSlug,
+  frontMatterSlug,
```

Comment:
Is this a breaking change?

Reply:
No, we don't expose this function to the end users.